app/Http/Controllers/Auth/LoginController.php

```diff
@@ -67,7 +67,7 @@ protected function verifyLogin(Request $request) {
 
     protected function vatusaAuth($resourceOwner, $accessToken) {
         $client = new Client();
-        $result = $client->request('GET', 'https://api.vatusa.net/v2/user/' . $resourceOwner->data->cid . '?apikey=' . Config::get('vatusa.api_key'), ['http_errors' => false]);
+        $result = $client->request('GET', Config::get('vatusa.base').'/v2/user/' . $resourceOwner->data->cid . '?apikey=' . Config::get('vatusa.api_key'), ['http_errors' => false]);
```

Comment:
See comment above - move to separate issue.

Reply:
Removed